apps/prairielearn/src/lib/question-render.ts

```diff
@@ -636,12 +643,10 @@ export async function renderPanelsForSubmission({
   user: User;
   urlPrefix: string;
   questionContext: QuestionContext;
+  questionRenderContext?: QuestionRenderContext;
   authorizedEdit: boolean;
   renderScorePanels: boolean;
   groupRolePermissions: { can_view: boolean; can_submit: boolean } | null;
-  localsOverrides?: {
-    manualGradingInterface?: boolean;
-  };
```

Comment:
This felt like a hack when I wrote it, `questionRenderContext` is what I was looking for but hadn't quite arrived at yet!